### ISSUE TITLE
How to use permission from model_has_permissions only?

### ISSUE THREAD

**Issue by ThunderBirdsX3** [spatie/laravel-permission#1632]
My system can assign multiple role, And each Role has default permissions.

But I let user can choose, Which permission from Role they want to assign.

So, How to give multiple role to user, But use permissions only from `model_has_permissions` not `role_has_permissions`.

**Comment by ThunderBirdsX3** [comment]
Should I set this code directly in User Model?
```
    /**
     * Determine if the model has, via roles, the given permission.
     *
     * @param \Spatie\Permission\Contracts\Permission $permission
     *
     * @return bool
     */
    protected function hasPermissionViaRole(Permission $permission): bool
    {
        return false;
    }
```

**Comment by drbyte** [comment]
Closed in #1633
